[2021-08-10 18:59:57,621] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask3 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 18:59:57,626] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask3 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 18:59:57,626] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:59:57,626] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-08-10 18:59:57,626] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:59:57,703] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): Covidtask3> on 2021-08-10T13:14:01.532369+00:00
[2021-08-10 18:59:57,707] {standard_task_runner.py:52} INFO - Started process 35725 to run task
[2021-08-10 18:59:57,713] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Covid', 'Covidtask3', '2021-08-10T13:14:01.532369+00:00', '--job-id', '359', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpousfpb12', '--error-file', '/tmp/tmph49xhujd']
[2021-08-10 18:59:57,714] {standard_task_runner.py:77} INFO - Job 359: Subtask Covidtask3
[2021-08-10 18:59:57,824] {logging_mixin.py:104} INFO - Running <TaskInstance: Covid.Covidtask3 2021-08-10T13:14:01.532369+00:00 [running]> on host deepika-Inspiron-5570
[2021-08-10 18:59:58,042] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Covid
AIRFLOW_CTX_TASK_ID=Covidtask3
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T13:14:01.532369+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T13:14:01.532369+00:00
[2021-08-10 19:00:03,539] {python.py:151} INFO - Done. Returned value was: None
[2021-08-10 19:00:03,541] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Covid, task_id=Covidtask3, execution_date=20210810T131401, start_date=20210810T131457, end_date=20210810T131503
[2021-08-10 19:00:03,728] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-10 19:00:03,768] {local_task_job.py:151} INFO - Task exited with return code 0
